docker/pg_tests/Dockerfile_several_delta_backups_test

```diff
@@ -0,0 +1,24 @@
+
+FROM wal-g/golang:latest as build
+
+WORKDIR /go/src/github.com/wal-g/wal-g
+
+RUN apt-get update &&     apt-get install --yes --no-install-recommends --no-install-suggests     liblzo2-dev
+
+COPY vendor/ vendor/
+COPY internal/ internal/
+COPY cmd/ cmd/
+COPY main/ main/
+COPY utility/ utility/
+
+RUN cd main/pg &&     go build -race -o wal-g -tags lzo -ldflags "-s -w -X main.BuildDate=`date -u +%Y.%m.%d_%H:%M:%S`"
+
+FROM wal-g/pg:latest
+
+COPY --from=build /go/src/github.com/wal-g/wal-g/main/pg/wal-g /usr/bin
+
+COPY docker/pg_tests/scripts/ /
```

Comment:
How you use this line?

Reply:
in that test "./scripts/drop_pg.sh" is used , but in code I can't find that link
That's why I decide to copy scripts to ./